examples/language-modeling/README.md

```diff
@@ -157,15 +157,15 @@ This example has been validated with the following DeepSpeed ZeRO-2 config: http
 
 ## RoBERTa/BERT/DistilBERT and masked language modeling
 
-The following examples fine-tune RoBERTa on WikiText-2. Here too, we're using the raw WikiText-2. The loss is different as BERT/RoBERTa have a bidirectional mechanism; we're therefore using the same loss that was used during their pre-training: masked language modeling.
+The following examples fine-tune RoBERTa on WikiText-2 with eager mode and torch.compile enabled. Here too, we're using the raw WikiText-2. The loss is different as BERT/RoBERTa have a bidirectional mechanism; we're therefore using the same loss that was used during their pre-training: masked language modeling.
 Following the RoBERTa paper, we use dynamic masking rather than static masking. The model may, therefore,
 converge slightly slower (over-fitting takes more epochs).
 
 
 ### Single-card Training
 
 ```bash
-python run_mlm.py \
+PT_HPU_LAZY_MODE=0 PT_ENABLE_INT64_SUPPORT=1 python run_mlm.py \
     --model_name_or_path roberta-base \
```

Comment:
Roberta Base for now is supported in lazy mode. Roberta Large to be supported in torch.compile

Reply:
Fixed in new commit "Changes after review". Rebased to latest habana-main.